app/src/components/SectionDisplay.tsx

```diff
@@ -28,12 +28,12 @@ export default class SectionDisplay extends Component<Props, States> {
         const section = this.props.section;
 
         // TODO: refactor
-        let allTimes: React.JSX.Element[] = API_Days.map((day) =>
+        let allTimes: React.JSX.Element[] = API_Days.map((day, i) =>
             section.meetings[day].length > 0 ? (
-                <div className="mx-1">
```

Comment:
You cannot just add an index. An index is useless for what React wants to take advantage of. React wants to be able to distinguish between two items in a list but also between items after a state change. In other words, if you have an element with `key={4}` in one state, then a state change occurs that causes a list to rerender, React expects that the item with `key={4}` in the next state will be the same as the previous state. This is guaranteed if `schedules` array changes.

See: https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

Reply:
I don't think omitting the key prop for items in a list is a recommended practice. For this particular section, API_Days doesn't seem to change, so I think we can use the index as the key. I think the same goes for `section.meetings[day]`.